src/main/zapHomeFiles/hud/serviceworker.js

```diff
@@ -56,9 +59,9 @@ localforage.setItem("tools", [])
 	})
 	.catch(utils.errorHandler);
 
-const onInstall = event => {
-	utils.log(LOG_INFO, 'serviceworker.install', 'Installing...');
+initWebSockets();
```

Comment:
does the placement of this call above `onInstall` with the function defintion below it matter? From my understanding it shouldn't, but thought I'd check. It would be nice if `onInstall` and `onActivate` were next to each other because they're related functions.

Reply:
No, but can easily move it.